app/serializers/user.js

```diff
@@ -0,0 +1,16 @@
+import DS from 'ember-data';
+
+export default DS.RESTSerializer.extend({
+    attrs: {
+        id: 'user_id',
+        username: 'username',
```

Comment:
Please remove this line because 'username' is equals to 'username', this line is unnecessary